### ISSUE TITLE
[jlaunch] Database Version not sufficient

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-websites#1914]
I am trying to install the latest alpha of Joomla 5 but it does not work because of the wrong database version.

**Comment by pe7er** [comment]
Yes, Joomla 5 needs at least MySQL 8 + PHP 8.1. 
MySQL 5.7 is End-Of-Life on 31 Oct 2023.

See: https://developer.joomla.org/news/904-joomla-5-0-alpha-2-new-ideas-added-to-joomla-5.html
> Update the system requirements to PHP 8.1 & MySQL 8.0.13+, MariaDB 10.4+ and PostgreSQL 12+

**Comment by HLeithner** [comment]
@Pawel-Panek does your platform already support mysql 8.0.13 or Mariadb 10.4 ?

**Comment by Pawel-Panek** [comment]
Yes, we noticed that too. MySQL 8.0 deployment is already in progress. It should be done this week.

**Comment by coolcat-creations** [comment]
I am still not Able to install Joomla 5 beta 1  although the banner says "nightly builds" please can you fix it @Pawel-Panek ?

**Comment by Pawel-Panek** [comment]
Finally it seems to be provisioning correctly. The nightly build from LJO is now creating a Joomla 5.0.0 rc2 as this is the one we found to be the most recent. It works with MySQL 8 as required by the installer.

**Comment by HLeithner** [comment]
@coolcat-creations can you please confirm and close the issue here?

**Comment by coolcat-creations** [comment]
I confirm that I successfully could install Joomla 5. 
BUT following is wrong:
- Certificate does not work
- If you announce that it's the nightly build it should be the nightly build and not the rc2. So either you have to change the text on the button or the version.
Also please can 3.10 be removed as an option, its not supported anymore I dont see any reason why people should have the possibility to install it through joomla.com?
